scalding-commons/src/main/java/com/twitter/scalding/commons/scheme/KeyValueByteScheme.java

```diff
@@ -24,6 +26,11 @@ public KeyValueByteScheme(Fields fields) {
     super(fields, BytesWritable.class, BytesWritable.class);
   }
 
+  @Override
+  public void sourceConfInit(FlowProcess<JobConf> flowProcess, Tap<JobConf, RecordReader, OutputCollector> tap, JobConf conf) {
+    conf.setInputFormat(CombineSequenceFileInputFormat.class);
```

Comment:
I believe that it should be opt-in and affect not only VKVS.

And also, it better to moves to hadoop implementation instead elephantbird.